### PR TITLE
Remove trailing space from BoundDecisionDagNode

### PR DESCRIPTION
**Pull request by sharwell**
Observed this while working on formatter performance investigations.

### CODE REVIEW

File: src/Compilers/CSharp/Portable/Generated/BoundNodes.xml.Generated.cs [hunk 1]
```diff
@@ -9005,7 +8996,11 @@ internal abstract partial class BoundTreeWalker: BoundTreeVisitor
             this.VisitList(node.SwitchArms);
             return null;
         }
-        public override BoundNode? VisitDecisionDag(BoundDecisionDag node) => null;
+        public override BoundNode? VisitDecisionDag(BoundDecisionDag node)
+        {
+            this.Visit(node.RootNode);
+            return null;
+        }
```
Comment (cston) [inline src/Compilers/CSharp/Portable/Generated/BoundNodes.xml.Generated.cs:9003]: Are these code changes the result of removing the space? If so, we should throw an exception in the `BoundNode` generator if the name fields contain spaces.

Comment (gafter) [reply src/Compilers/CSharp/Portable/Generated/BoundNodes.xml.Generated.cs:9003]: Agreed. However, this change is harmless as we don't use the visitor mechanism for decision dags or its nodes.

Comment (sharwell) [reply src/Compilers/CSharp/Portable/Generated/BoundNodes.xml.Generated.cs:9003]: > Are these code changes the result of removing the space?

Yes

Comment (cston) [reply src/Compilers/CSharp/Portable/Generated/BoundNodes.xml.Generated.cs:9003]: [https://github.com/dotnet/roslyn/issues/43986](https://github.com/dotnet/roslyn/issues/43986)

---
In reply to: [420325700](https://github.com/dotnet/roslyn/pull/43982#discussion_r420325700) [](ancestors = 420325700)

**Pull request merged**